pages/visitors/[id]/start.js

```diff
@@ -49,4 +54,17 @@ const Start = () => {
   );
 };
 
+export const getServerSideProps = propsWithContainer(
+  async ({ query, container }) => {
+    const { id, callPassword } = query;
+    const callId = id;
+    const queryPassword = callPassword ? callPassword : "";
+    const { scheduledCall, error } = await retrieveVisitByCallId(container)(
+      callId
+    );
+    const dbPassword = scheduledCall.callPassword;
+    return { props: { callId, error, queryPassword, dbPassword } };
+  }
```

Comment:
Technically this would provide the end user with the real password if they looked in the page source

Reply:
I would rather you redirect the user immediately if the password is incorrect. Ie do the check in this function (which runs server side) and if they fail then redirect them to `/error` similar to 
https://github.com/madetech/nhs-virtual-visit/blob/b9c428d21cf0be5cada288211c3315244dfc019a/pages/index.js#L6-L9

Reply:
Copied this pattern 👍